### PR TITLE
Support default scope in regional cloud environments

### PR DESCRIPTION
**Pull request by RakhithaRR**
## Purpose
This PR supports the default scope in Azure AD in region specific cloud environments. The value `https://graph.microsoft.com/.default` is not valid if the Azure Graph API is in a separate National Cloud deployment.

Related internal: https://github.com/wso2-enterprise/wso2-apim-internal/issues/5485

Related issue: https://github.com/wso2/api-manager/issues/2456

### CODE REVIEW

**Review by tharikaGitHub — APPROVED** [review]

File: .gitignore [hunk 1]
```diff
@@ -25,4 +25,5 @@ target/
 .vscode/
 .mvn/
 mvnw
-.idea
\ No newline at end of file
+.idea
```
Comment (shilmyhasan) [inline .gitignore:28]: is this change required?

Comment (RakhithaRR) [reply .gitignore:28]: yes, otherwise someone might commit the useless files by mistake

**Review by dushaniw — APPROVED** [review]
